testsuite/tests/lexical_envs/add_to_env_mult_dests/main.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import, division, print_function
+
+print('main.py: Running...')
+
+
+import sys
+
+import libfoolang
+
+
+ctx = libfoolang.AnalysisContext()
+u = ctx.get_from_file('input')
+if u.diagnostics:
+    for d in u.diagnostics:
+        print(d)
+    sys.exit(1)
+
+try:
+    u.populate_lexical_env()
+    for i in u.root.findall(libfoolang.Id):
+        print("Scope for {}: {}".format(i, i.p_get_scope))
+except libfoolang.PropertyError as exc:
+    print('   Got an exception: {}'.format(str(exc).strip()))
+else:
+    print('   All good...')
```

Comment:
Given that you don’t except any exception here, the except/else clauses could be removed (and thus keep only the content of the try clause)

Reply:
Right!

Reply:
Thanks! I’ll let you adjust this PR: LGTM otherwise.